src/modules/Tab/Tab.js

```diff
@@ -110,9 +110,13 @@ class Tab extends Component {
   }
 
   renderMenu() {
-    const { menu, panes } = this.props
+    const { menu, panes, menuPosition } = this.props
     const { activeIndex } = this.state
 
+    if (menu.tabular === true) {
+      menu.tabular = menuPosition === 'left' || 'right'
+    }
```

Comment:
since there is no support for value `'left'` in `menu.tabular`, we cannot set `menu.tabular` as `'left'` AND `menuPosition` as `'right'`

Reply:
this way we can specify `menu.tabular` as `true` if we just want to infer it from `menuPosition`
and as `left` or `right` if we want to specify ourselves